### PR TITLE
docs: Update overrideNative information

### PR DESCRIPTION
**Pull request by mister-ben**
Updates the `overrideNative` section, since that's a frequent cause of confusion, or not using it is a frequent cause of issue with bad native HLS implementations.

Removes using it a source option, because that doesn't work.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -316,25 +324,21 @@ is true, if the platform supports Media Source Extensions
...
 
-// via the source
-var player = videojs('playerId');
-
-player.src({
-  src: 'https://example.com/index.m3u8',
-  type: 'application/x-mpegURL',
-  overrideNative: true
-});
-```
```
Comment (gkatsev) [inline README.md]: We need to close the code block.

**Comment by gkatsev** [comment]
Hm... using it as a source option *should* work.

**Comment by mister-ben** [comment]
Only `options.hls.overrideNative` is considered in canPlayType https://github.com/videojs/http-streaming/blob/e59428a0f16e6ff44f5a327a9fcc137a3c13d915/src/videojs-http-streaming.js#L817-L824

File: README.md [hunk 2]
```diff
@@ -316,26 +324,23 @@ is true, if the platform supports Media Source Extensions
 videojs-http-streaming will take over HLS playback to provide a more
 consistent experience.
 
+If `overrideNative` is set to `true`, then `nativeAudioTracks` and `nativeVideoTracks` should be set to `false`.
```
[the remark this answers is not in the document]
Comment (gkatsev) [reply README.md]: Just tried locally and seems like it isn't needed. However, it's possible that currently, setting `nativeTextTracks` to false is necessary due to how we use the id3 track in VHS.

[the remark this answers is not in the document]
Comment (gesinger) [reply README.md]: I think if we remove this line from the README changes, we should be good to merge then. @mister-ben thoughts?

**Pull request merged**